cpp/ps0000/ps0000_soln.cpp

```diff
@@ -1,12 +1,10 @@
-﻿
-#include <iostream>
+﻿#include <iostream> // 2/2
 
 using namespace std;
 
-int main()
+int main() // 1/1
 {
-	cout << "Hello, World!\n";
-
-		return 0;
+	cout << "Hello, World!\n"; // 4/4
+	return 0; // 1/1
 }
-
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit